packages/toolkit/src/view/recipe-editor/popovers/PipelineNamePopover.tsx

```diff
@@ -72,13 +77,16 @@ export const PipelineNamePopover = ({
     });
   }, [routeInfo.isSuccess, routeInfo.data.resourceId, form, pipelineIsPublic]);
 
+  const formattedPipelineId = formatResourceId(form.watch("id"), "p");
```

Comment:
Should we move `p`, `m`, `c` etc. to a separate constants file?

Reply:
Oh that is a good suggestion, I will do that 🤝